infrastructure/src/main/resources/short_lease_lock_config.properties

```diff
@@ -0,0 +1,7 @@
+lock_group=Log_Replication_Group
+lock_name=Log_Replication_Lock
+lock_lease_duration_seconds=5
+lock_monitor_duration_seconds=60
+lock_duration_between_lease_checks_seconds=60
+lock_duration_between_lease_renewals_seconds=60
```

Comment:
Just to confirm, the intention of this config is that the lease is acquired for a short time, and lost right away?  as the monitor and renewal duration are still big compared the lease time.

Reply:
This is a config that is used in the integration tests if we run LRs as threads. It does not run by default. I double-checked on how we are doing it in master and fixed it a bit.